### ISSUE TITLE
 iptv pvr simple not load channel 

### ISSUE THREAD

**Issue by Vikings77** [kodi-pvr/pvr.iptvsimple#596]
hi
ihave a m3u file but not load in kodi pvr simple
pvr manager is starting up 0 %

i test windows 10 kodi 19.3 matrix and
kodi-18-9 leia
in vlc doesn't problem

![150949035-2c295be2-c90f-4223-b614-c40a9c674cde](https://user-images.githubusercontent.com/98409167/151007365-a68f5ee5-5657-431c-b58e-099384a2809c.jpg)

**Comment by phunkyfish** [comment]
How large is the file?

**Comment by Vikings77** [comment]
File size 49mb 
m3u have a TV Channel and vod

Once, two days ago the channels were loaded ,
 but now no load

im unistall and install kodi 
unistall and install pvr simple add-on 
Could not be resolved

**Comment by phunkyfish** [comment]
That is an absolutely monstrous sized file. There must be millions of entries in total.try to start kodi and leave it there for several hours and see if it loads. If you then look into he log you should see a playlist loaded timing. See what that is altogether. Do you also have an EPG for those channels/VOD?

**Comment by Vikings77** [comment]
not epg sir

**Comment by phunkyfish** [comment]
Ok, let me know what you find in your debug log after several hours.

**Comment by Ashrafmansy** [comment]
I have the same issue with one channel in the m3u file. It was working fine but suddenly I got the same error in the screenshot and it stopped working since then

**Comment by phunkyfish** [comment]
> I have the same issue with one channel in the m3u file. It was working fine but suddenly I got the same error in the screenshot and it stopped working since then

How do you know it’s one channel causing this if you never get passed the above screenshot?

**Comment by Vipskywalker27** [comment]
Hi , and thanks for all this fantastic work , I have a question my PVR IPTV simple client work great but yesterday dont load nothing , my playlist is only 50 chanels and unistall and reinstall the pvr and i cant fix it , could help me with that ? thanks !

**Comment by phunkyfish** [comment]
> Hi , and thanks for all this fantastic work , I have a question my PVR IPTV simple client work great but yesterday dont load nothing , my playlist is only 50 chanels and unistall and reinstall the pvr and i cant fix it , could help me with that ? thanks !

Please create a new issue and add a full debug log.

**Comment by Vipskywalker27** [comment]
ok , im ready !

**Comment by demesg** [comment]
any news on this issue ?

**Comment by phunkyfish** [comment]
> any news on this issue ?

No, the thread went cold. If someone can supply an example I can use to reproduce I’m happy to take a look.

**Comment by demesg** [comment]
https://paste.kodi.tv/otogesatow.kodi

**Comment by phunkyfish** [comment]
> https://paste.kodi.tv/otogesatow.kodi

What exactly is captured in this log?

**Comment by aamaamilan** [comment]
same issue here.
i solved this by going back to Kodi 17 it's working normal.

**Comment by phunkyfish** [comment]
> same issue here.
> i solved this by going back to Kodi 17 it's working normal.

Please provide a full debug log capturing the problem.

**Comment by SwappoX** [comment]
Same issue here, loading channels from client stuck at 0%.
Using m3u with free channels works fine but with usename and password in url never gets past 0%

Running Kodi 19.4 Matrix

**Comment by phunkyfish** [comment]
Unless anyone can provide a full debug log capturing the problem I intend to close this issue. 

In 3 months no one has posted one so I can’t help I’m afraid.

**Comment by SwappoX** [comment]
https://pastebin.com/RvJeh4Jm

**Comment by SwappoX** [comment]
https://pastebin.com/RvJeh4Jm

Username and password is removed in logfile

**Comment by phunkyfish** [comment]
> Same issue here, loading channels from client stuck at 0%. Using m3u with free channels works fine but with usename and password in url never gets past 0%
> 
> Running Kodi 19.4 Matrix

Thanks. You appear to have an error in your URL. Unless you modified the URL in some way already. Regardless it’s giving an error.

`2022-03-27 12:34:25.291 T:941     ERROR <general>: CCurlFile::Open failed with code 404 for http://lol.bz:2095/get.php?`

If you’d like to share your provider details privately on gitter.im (I have the same username there) I can try and figure out why it’s not working for you.

**Comment by SwappoX** [comment]
Yes, I did notice that 404 line and the other.
Thing is that the url is working with other clients as well as clicking on it.

I'll try to get in touch with you via glitter.im

**Comment by praveenrai2k-rgb** [comment]
To fix this issue download the Demo PVR Client first and then download IPTV SIMPLE CLIENT.
